### PR TITLE
Code formatted.

### PR DESCRIPTION
**Pull request by developer-harisubedi**

### CODE REVIEW

File: src/main/java/np/com/pantbinod/springbootfinaldemo/common/BaseConverter.java [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 public class BaseConverter<E,D> implements ISingleEntryConverter<E,D>, IListEntryConverter<E,D>{
    
+   
+
```
Comment (devbinod) [inline src/main/java/np/com/pantbinod/springbootfinaldemo/common/BaseConverter.java:10]: Please remove spaces

**Review by devbinod — CHANGES_REQUESTED** [review]
https://github.com/devbinod/springbootfinaldemo/pull/6/files#diff-c043924ae928d28e4d69f13b405ac296R10

**Review by devbinod — APPROVED** [review]
Looks great!!! :(

**Review by arjunghimire5 — APPROVED** [review]
great

**Pull request merged**